functions/packages.bash

```diff
@@ -18,16 +18,14 @@ samba_setup() {
   cond_redirect cp "${BASEDIR:-/opt/openhabian}"/includes/smb.conf /etc/samba/smb.conf
   cond_echo "\nWriting authentication data to openHABian default... "
   if ! smbpasswd -e "${username:-openhabian}" &> /dev/null; then
-    # shellcheck disable=SC2154
-    (echo "$userpw"; echo "$userpw") | smbpasswd -s -a "${username:-openhabian}" &> /dev/null
+    (echo "${userpw:-openhabian}"; echo "${userpw:-openhabian}") | smbpasswd -s -a "${username:-openhabian}" &> /dev/null
   fi
   echo "OK"
 
   echo -n "$(timestamp) [openHABian] Setting up Samba service... "
-  sed -iE '/PIDFile/d; /NotifyAccess/ a PIDFile=smbd.pid\nRuntimeDirectory=samba' $serviceFile
-  cond_redirect systemctl -q daemon-reload &> /dev/null
-  if ! cond_redirect systemctl enable smbd.service; then echo "FAILED (enable service)"; return 1; fi
-  if cond_redirect systemctl restart smbd.service; then echo "OK"; else echo "FAILED (restart service)"; return 1; fi
+  if ! cond_redirect sed -i -E '/PIDFile/d; /NotifyAccess/ a PIDFile=smbd.pid\nRuntimeDirectory=samba' "$serviceFile"; then echo "FAILED"; return 1; fi
```

Comment:
we eventually might need a mkdir here (for RuntimeDirectory?), see #1112

Reply:
Do you want me just to go ahead and add it now?

Reply:
yeah why not. Attention though, this is just an assumption, I'm not sure this is the reason for the error message but you'll see in the testbuild won't you

Reply:
yep

Reply:
added

Reply:
and did it fix the error?